website/content/docs/configuration/events/configure-events.mdx

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+page_title: Configure events
+description: >-
+  Configure log event sinks for controllers and workers to monitor Boundary resources and actions.
+---
+
+# Configure event logging
+
+The [`events`](/boundary/docs/configuration/events) stanza lets you log system information in a well-defined, structured format that gives you visibility into emitted events.
+
+You configure emitted events and sinks in the configuration file for any [controller](/boundary/docs/configuration/controller) or [worker](/boundary/docs/configuration/worker) server.
+Complete the following steps to configure events:
+
+1. Open the configuration file for the controller or worker server that you want to emit events.
+1. Edit the `events` stanza in the configuration file.
```

Comment:
I think this should be item 2

Reply:
Ah yeah, our Markdown standard is a little wonky. That still renders as numbers 2 and 3. It should be OK in the preview deployment:

<img width="629" alt="image" src="https://github.com/user-attachments/assets/786f65f2-2002-4375-855e-430a52f8791d">